setup.py

```diff
@@ -19,13 +19,13 @@
     """
 setup(
     name="dlt_meta",
-    version="0.0.1",
+    version="0.0.2",
     python_requires=">=3.8",
-    setup_requires=["wheel>=0.37.1,<0.38"],
+    setup_requires=["wheel>=0.37.1,<=0.40.0"],
     install_requires=INSTALL_REQUIRES,
     extras_require={"dev": DEV_REQUIREMENTS, "IT": IT_REQUIREMENTS},
     author="Ravi Gawai",
-    author_email="ravi@databricks.com",
```

Comment:
should we have a public email, like, `databrickslabs@databricks.com` ?

Reply:
good catch! will change this